README.md

```diff
@@ -45,21 +45,22 @@ pipeline {
     agent any
     triggers{
         bitbucketpr(projectPath:'<BIT_BUCKET_PATH>',
-        cron:'H/15 * * * *',
-        credentialsId:'',
-        username:'',
-        password:'',
-        repositoryOwner:'',
-        repositoryName:'',
-        branchesFilter:'',
-        branchesFilterBySCMIncludes:false,
-        ciKey:'',
-        ciName:'',
-        ciSkipPhrases:'',
-        checkDestinationCommit:false,
-        approveIfSuccess:false,
-        cancelOutdatedJobs:true,
-        commentTrigger:'')
+            cron: 'H/15 * * * *',
+            credentialsId: '',
+            username: '',
+            password: '',
+            repositoryOwner: '',
+            repositoryName: '',
+            branchesFilter: '',
+            branchesFilterBySCMIncludes: false,
+            ciKey: '',
+            ciName: '',
+            ciSkipPhrases: '',
+            checkDestinationCommit: false,
+            approveIfSuccess: false,
+            cancelOutdatedJobs: true,
+            buildChronologically: true,
```

Comment:
Here, `buildCrohologically` is set to true.

Reply:
If you were to change this one line to false, the rest of the logic would match up to the documentation. This is the only remaining change that I think this needs.

Or, I'm mis-interpreting the code -vs- behavior. In that case, feel free to let me know why I'm wrong ;-)

Reply:
I believe, that commented code snippet is not a part of documentation. As far as I can understand it's just an example of easy way to configure pipeline job.

You can see the same situation with setting `cancelOutdatedJobs`: here it's value is set to `true`, but the real default value is `false` and defined at `config.jelly` file as
```
<f:entry title="Cancel outdated jobs?" field="cancelOutdatedJobs">
  <f:checkbox default="false"/>
</f:entry>
```
So, in this case if user just copy config from `Readme.md` file the setting's value will be `true`. If user enable build trigger in job's settings, the value of this parameter will be `false` (unchecked checkbox).

Using proposed value for `buildChronologically` in `Readme.md` is more universal and build process will be the same with no difference you have a lot of frequently changing PRs or not.
That's why I've set this value to `true` in example.
I guess, the same reason was for `cancelOutdatedJobs` - in most cases it have no sense to continue current build if PR has been updated.

Reply:
ok - I have no more changes for you.